gateway/api_loader.go

```diff
@@ -676,6 +678,29 @@ type generalStores struct {
 	redisStore, redisOrgStore, healthStore, rpcAuthStore, rpcOrgStore storage.Handler
 }
 
+func loadGraphQLPlayground(router *mux.Router) {
```

Comment:
Needs to be decided where to live, I put it now temporarily.

Reply:
I implemented this as another PR to fix the sprint5 task separately. I will send the PR after this is merged. Let's keep this in the root playground level for this PR.